FLT/DivisionAlgebra/Finiteness.lean

```diff
@@ -135,6 +135,15 @@ local instance : IsModuleTopology (FiniteAdeleRing (𝓞 K) K) (D ⊗[K] (Finite
 
 variable [FiniteDimensional K D]
 
+-- Instance to help speed up instance synthesis
```

Comment:
I wasn't able to find a better solution to instances timing out here

Reply:
I think it's fine to make instance shortcuts. We're pushing typeclass inference to the limits it seems with these arguments. As long as there are no diamonds I think shortcuts are fine.